ext/InfiniteExaModelsMadNLP.jl

```diff
@@ -56,6 +57,15 @@ function InfiniteExaModels.resolve(
     return MadNLP.solve!(backend.model, solver; sol_options...)
 end
 
+function InfiniteExaModels.warmstart_backend(
+    backend::InfiniteExaModels.ExaTranscriptionBackend,
+    solver::MadNLP.MadNLPSolver
+    )
+    results = backend.results
+    copyto!(NLPModels.get_x0(backend.model), results.solution)
+    return
+end
+
```

Comment:
This should be the default behavior in `infiniteopt_backend` (not specific to MadNLP). So I think this should be removed.